ci_watson/hst_helpers.py

```diff
@@ -128,15 +138,39 @@ def download_crds(refname, timeout=30, verbose=False):
     if refdir is None:
         raise ValueError('Unknown HTTP destination for {}'.format(refname))
 
-    from ci_watson.artifactory_helpers import check_url, _download
+    # need to insure CRDS has been cached locally
+    os.environ['CRDS_SERVER_URL'] = CRDS_SERVER_URL
+    os.environ['CRDS_PATH']='./'
```

Comment:
If you ever run the test locally, is this behavior desirable? Won't this download a bunch of stuff into the working dir?

Reply:
Actually, that is the point of using the 'crds sync' call below; namely, only those files which are requested get downloaded whether it is local or remote.  I ran this on my Windows system at home and only downloaded the one reference file that was requested without getting any other CRDS file.  These definitions, though, are required in order to tell CRDS what server to use and that it should only download the files into the local directory (to start with anyway).

Reply:
This change was made in b68f309.  It was also tested on my desktop Linux workstation at work (freyja) and when 'iref' was defined to point to the local directory, it only copied the desired reference file as requested, while it did not copy anything when 'iref' was pointing to the already available local online cache (as before).  It also still works just fine under Windows as well.

Reply:
Does hstcal even run on Windows? 👀

Reply:
Just don't want to get into the situation where dev runs this locally, downloads a bunch of files into the working dir (which might be source dir), and then accidentally `git add` all of them into version control.

> make this definition dependent on whether or not it was already defined by the user. Would that be safer?

Yes, I think so.

Reply:
My tests have been run primarily on Drizzlepac under Windows, not hstcal, as I have not tried to build hstcal C code under Windows.  This definition of the URL was made optional under b68f309 and worked as expected only downloading the requested single reference file on freyja (STScI Linux desktop).